### DISCUSSION TITLE
Run geoclimate only for a part of the city

### DISCUSSION THREAD

**Discussion by lucianaswf** [orbisgis/geoclimate#894] [Q&A]
Hello.
I want to run Geoclimate for São Paulo, Brazil. 
Since the city is too large (more than 1500km2) I need to do it in smaller parts (around 400km2).
Preferably, I'd like to use the city's official divisions (East, West Zones, etc).
Do you know how can I specify this when running GeoClimate?
I want to use the command Line Interface with OSM.
Thank you

**Comment by j3r3m1** [comment]
Hi @lucianaswf 
You can have a try with the name of each district in the Nominatim API: https://nominatim.openstreetmap.org/ui/search.html?q=sao+paulo

The idea is to check that the name you will type fits with the zone you would like. If not, the easieast will be to use bbox that you can for example roughly define here: http://bboxfinder.com/#0.000000,0.000000,0.000000,0.000000 (draw a rectangle, click on it and collect the bounding box coordinates in the URL)

**Comment by lucianaswf** [comment]
Thank you @j3r3m1 !

Some neighborhoods are well nominated, but other no. Is it possible to create new nominations?

Best regards!

**Reply by j3r3m1** [reply]
I suppose yes. @ebocher or @simogeo might tell you more about it.

But first I propose you test GeoClimate with the neighborhoods that are well nominated to see if everything is fine ?

**Reply by lucianaswf** [reply]
Yes! I will do it.
Thank you

**Reply by lucianaswf** [reply]
It worked with the name of the neighborhood, but not with the coordinates. 
I tested with just 2 coordinates of opposite vertices and with all 4 coordinates. Always in the EPSG:4326 projection.

The config file looked like this: 

"input": {
        "locations": [[-46.678677,-23.562886,-46.589241,-23.504024]]
    },


Then, the error I got was this: 

11:34:10.214 [main] INFO class org.orbisgis.geoclimate.osm.OSM -- 4 osm areas will be processed
11:34:10.252 [main] ERROR class org.orbisgis.geoclimate.osm.OSM -- Cannot find an area from the location -46.678677
11:34:10.254 [main] ERROR class org.orbisgis.geoclimate.osm.OSM -- Cannot find an area from the location -23.562886
11:34:10.255 [main] ERROR class org.orbisgis.geoclimate.osm.OSM -- Cannot find an area from the location -46.589241
11:34:10.256 [main] ERROR class org.orbisgis.geoclimate.osm.OSM -- Cannot find an area from the location -23.504024
11:34:10.314 [main] ERROR org.orbisgis.data.jdbc.JdbcTable -- Cannot save the file : C:\Users\biota\AppData\Local\Temp\\logzones.geojson
Cannot execute the OSM workflow

Any suggestions?

**Reply by j3r3m1** [reply]
I have updated your location directly in your message.
The location is made for selecting several areas at the same time (each of them being processed and saved separately however), so you need an extra bracket (you may put several bounding box in a list)

**Reply by lucianaswf** [reply]
I'm sorry, but I think there's a projection problem because the results for this area are from a zone in the middle of the ocean. I extracted the coordinates of my area in EPSG 4326, is that the correct projection?
Thank you very much for your help.

**Reply by j3r3m1** [reply]
No worries the reason is that latitude should be used first:
[[-23.562886,-46.678677,-23.504024,-46.589241]]

**Comment by lucianaswf** [comment]
I have well divided São Paulo in 10x10km areas and I've been able to properly run Geoclimate for those areas. However I got a warning in the script for all areas:  _**[main] WARN class org.orbisgis.geoclimate.Geoindicators --  The column pop doesn't exist or should be numeric.**_
Is this an issue that I should be worried about? Can this warning compromise the results?
Thank you for your help!

**Reply by j3r3m1** [reply]
No worries it will not affect your results. This warns you that no population was retrieved from the worldpop data. Thus you will not have population indicator but all other indicators will be fine.
